ibm_catalog.json

```diff
@@ -562,8 +562,189 @@
+                  "value" : "public"
+                }
+              ]
+            },
+            {
+              "key": "log_analysis_agent_custom_line_inclusion"
+            },
+            {
+              "key": "log_analysis_agent_custom_line_exclusion"
+            },
+            {
+              "key": "log_analysis_agent_name"
+            },
+            {
+              "key": "log_analysis_agent_namespace"
+            },
+            {
+              "key": "log_analysis_agent_tolerations"
+            },
+            {
+              "key": "cloud_monitoring_enabled"
+            },
+            {
+              "key": "cloud_monitoring_access_key"
+            },
+            {
+              "key": "cloud_monitoring_secret_name"
+            },
+            {
+              "key": "cloud_monitoring_instance_region"
+            },
+            {
+              "key": "cloud_monitoring_endpoint_type",
+              "options":[
+                {
+                  "displayname" : "private",
+                  "value" : "private"
+                },
+                {
+                  "displayname" : "public",
+                  "value" : "public"
+                }
+              ]
+            },
+            {
+              "key": "cloud_monitoring_metrics_filter"
+            },
+            {
+              "key": "cloud_monitoring_agent_tags"
+            },
+            {
+              "key": "cloud_monitoring_add_cluster_name"
+            },
+            {
+              "key": "cloud_monitoring_agent_name"
+            },
+            {
+              "key": "cloud_monitoring_agent_namespace"
+            },
+            {
+              "key": "cloud_monitoring_agent_tolerations"
+            },
+            {
+              "key": "logs_agent_enabled"
+            },
+            {
+              "key": "logs_agent_name"
+            },
+            {
+              "key": "logs_agent_namespace"
+            },
+            {
+              "key": "logs_agent_trusted_profile"
+            },
+            {
+              "key": "logs_agent_iam_api_key"
+            },
+            {
+              "key": "logs_agent_tolerations"
+            },
+            {
+              "key": "logs_agent_additional_log_source_paths"
+            },
+            {
+              "key": "logs_agent_exclude_log_source_paths"
+            },
+            {
+              "key": "logs_agent_selected_log_source_paths"
+            },
+            {
+              "key": "logs_agent_log_source_namespaces"
+            },
+            {
+              "key": "logs_agent_iam_mode"
+            },
+            {
+              "key": "logs_agent_iam_environment"
+            },
+            {
+              "key": "logs_agent_additional_metadata"
+            },
+            {
+              "key": "logs_agent_enable_scc"
+            },
+            {
+              "key": "cloud_logs_ingress_endpoint"
```

Comment:
I think you should force this to required since cloud logs is enabled by default. That way it will show in the `Required` tab in projects. Otherwise its lost down the bottom of the Optional tab, and will likely be missed. You can update the variable decription in the terraform code to mention its required value if `logs_agent_enabled` is set to true.